src/Core/SettingsChangesHistory.cpp

```diff
@@ -82,6 +82,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"use_skip_indexes_if_final_exact_mode", 0, 1, "Change in default value of setting"},
             {"allow_experimental_time_series_aggregate_functions", false, false, "New setting to enable experimental timeSeries* aggregate functions."},
             {"min_outstreams_per_resize_after_split", 0, 24, "New setting."},
+            {"count_matches_stop_at_empty_match", true, false, "New setting."},
```

Comment:
Oops, the new setting is still in the 25.6 section. The 25.7 setting would be the right one.

Two options to fix this:
- in a small separate PR
- add the `pr-must-backport` label and bring this PR into 25.6 (and only this version).